doc/local_auth.md

```diff
@@ -0,0 +1,69 @@
+# Setup local authentication for Assisted Migration
+
+AM uses jwt for authentication, hence, for local development, we need to setup a local authentication.
```

Comment:
I would say something like, `in order to use authentication for local develment` because by default we use none